code/modules/mob/living/silicon/robot/robot_helpers.dm

```diff
@@ -47,4 +47,4 @@
 	stored_module.loc = module
 	if(client)
 		client.screen -= stored_module
-	contents -= stored_module
\ No newline at end of file
+	contents -= stored_module
```

Comment:
You could just not commit this file if there's no changes. Doesn't matter I guess.

Reply:
Motherfu-